src/babel/transformation/transformers/minification/remove-react-prop-types.js

```diff
@@ -0,0 +1,14 @@
+import * as t from "../../../types";
+
+export var metadata = {
+  optional: true,
+  group: "builtin-setup"
+};
+
+export var Property = {
+  exit(node) {
+    if (node.key.name === 'propTypes') {
```

Comment:
@sebmck I don't know why, this code not working: `this.get("key").matchesPattern("propTypes")`